src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import type {ElementRef} from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+type Props = {
+    checked: boolean,
```

Comment:
If I think about it, it gets a bit confusing. What would you pass to the `onChange` callback the `checked` boolean or the string `value`?